### ISSUE TITLE
feedback

### ISSUE THREAD

**Issue by astrojetson8** [smka/homebridge-broadlink-sp#2]
Great job! I just tested this with SP 1 on my Mac, and it works.

I see following more of enhancements instead of issues:
- Is it possible to omit IP address or add mac address?
- Type should be "Outlet" instead of "Switch"

**Comment by astrojetson8** [comment]
also, shouldn't this be a platform instead of accessory?

**Comment by smka** [comment]
Hi! Thanks for feedback. 
- add mac address is not a problem
- type "Outlet" instead of "Switch" is good! I'll research this.
> shouldn't this be a platform instead of accessory? 

I don't know. What will it improve?

**Comment by astrojetson8** [comment]
thanks for getting back to me.

perhaps down the track, you could omit a need for IP or MAC address altogether? 

regarding changing from platform instead of accessory was schematic, and it wont improve anything.

**Comment by smka** [comment]
@astrojetson8 I've added mac address. You can use IP or MAC in config.json. Please, update and check it.

 "Outlet" instead of "Switch" will be in the next update, I think...

**Comment by astrojetson8** [comment]
@smka great, and thanks!

I've been using this plugin for a while. At times I see latency or non-responsiveness, and it just seems to be random. 

I'm running this on my Mac. Initially, I had this on Pi 0 W but I thought maybe a conflict was causing the latency and non-responsiveness. 
One the plugin on Pi is https://github.com/lprhodes/homebridge-broadlink-rm and it also uses broadlinkjs.

**Comment by smka** [comment]
@astrojetson8 non-responsiveness appears on the last version or in previous too?

**Comment by astrojetson8** [comment]
@smka in both version. 

```
[5/3/2017, 4:35:12 PM] Homebridge is running on port 51826.
[5/3/2017, 4:52:16 PM] [Kitchen Light] set SP state: true
[5/3/2017, 9:22:53 PM] [Kitchen Light] set SP state: true
[5/3/2017, 9:22:53 PM] [Kitchen Light] ON!
[5/3/2017, 9:25:33 PM] [Kitchen Light] set SP state: false
[5/3/2017, 9:48:09 PM] [Kitchen Light] set SP state: false
[5/3/2017, 9:49:30 PM] [Kitchen Light] set SP state: false
[5/3/2017, 9:49:31 PM] [Kitchen Light] OFF!
[5/3/2017, 11:20:50 PM] [Kitchen Light] set SP state: true
[5/3/2017, 11:20:50 PM] [Kitchen Light] ON!
[5/3/2017, 11:23:11 PM] [Kitchen Light] set SP state: false
[5/3/2017, 11:26:04 PM] [Kitchen Light] set SP state: false
[5/3/2017, 11:28:09 PM] [Kitchen Light] set SP state: false
[5/3/2017, 11:28:09 PM] [Kitchen Light] OFF!
[5/4/2017, 8:23:04 AM] [Kitchen Light] set SP state: true
[5/4/2017, 8:23:04 AM] [Kitchen Light] ON!
[5/4/2017, 8:23:14 AM] [Kitchen Light] set SP state: false
[5/4/2017, 8:23:15 AM] [Kitchen Light] OFF!
```

**Comment by smka** [comment]
@astrojetson8 I see...
Could you please run homebridge with only this plugin (broadlinkSP accessories only)?
If it will work better, then the problem may be a conflict with other plugin.

**Comment by astrojetson8** [comment]
@smka yes i'm running your plugin on mac without any other plugin. i suspect it's to do with polling.
one thing i didn't mention was i was using siri on watch and iphone. however, from home app i can see at times it says 'updating' which at times can be a long wait before it's connected.
